test/testmachinery/gardener/security/rbac.go

```diff
@@ -84,20 +86,36 @@ var _ = ginkgo.Describe("RBAC testing", func() {
 			framework.ExpectNoError(f.GardenClient.Client().Delete(ctx, serviceAccount))
 		}()
 
+		gardenVersion := f.GardenClient.Version()
+		gardenLess124, err := version.CheckVersionMeetsConstraint(gardenVersion, "< 1.24.0")
+		framework.ExpectNoError(err)
+
```

Comment:
Can you simplify this to always use the token request API?
The minimum supported garden version is 1.20 which should offer the token request API by default.